pkgs/applications/emulators/86box/default.nix

```diff
@@ -48,14 +49,26 @@ stdenv.mkDerivation rec {
     ++ lib.optional (!enableDynarec) "-DDYNAREC=OFF"
     ++ lib.optional (!unfreeEnableDiscord) "-DDISCORD=OFF";
 
-  postInstall = lib.optional stdenv.isLinux ''
+  roms = lib.optionalString unfreeEnableRoms (fetchFromGitHub {
+    owner = "86Box";
+    repo = "roms";
+    rev = "v${version}";
+    hash = "sha256-1HtoizO0QIGNjQTW0clzRp40h1ulw55+iTYz12UJSms=";
+  });
+
+  postInstall = ''
+    if [ ! -z "$roms" ]; then
+      mkdir -p $out/share/86Box
+      ln -s $roms $out/share/86Box/roms
+    fi
+  '' + (lib.optionalString stdenv.isLinux ''
     install -Dm644 -t $out/share/applications $src/src/unix/assets/net.86box.86Box.desktop
 
     for size in 48 64 72 96 128 192 256 512; do
       install -Dm644 -t $out/share/icons/hicolor/"$size"x"$size"/apps \
         $src/src/unix/assets/"$size"x"$size"/net.86box.86Box.png
     done;
-  '';
+  '');
```

Comment:
Unnecessary here - see [this comment](https://github.com/NixOS/nixpkgs/pull/288072/files#r1486462166)

Reply:
I added to make it explicit

Reply:
Removed